### DISCUSSION TITLE
✅ Mega Menu - Clickable icon and title together - [ED-13657]

### DISCUSSION THREAD

**Discussion by NAROLLES** [elementor/elementor#25112] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

only the title is clickable in widget pro Mega Menu

### Describe the solution you'd like.

I want a selector allowing the choice with icon clickable or not

### Describe alternatives you've considered.

custom css - no response in elementor group

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
Hi @NAROLLES!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro v3.22.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
